### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by lschatzkin**
initial styles for sec, with some simple templating changes

### CODE REVIEW

**Comment by wioux** [comment]
Great. A few things:

   * Add this to config/initializers/assets.rb so that `font-url` works:
         `Rails.application.config.assets.paths << Rails.root.join('app/assets/fonts')`
   * We should be consistent with whether or not included `.scss` files are prefixed with an underscore
   * The linter requires leading zeroes before a unit like `.5rem`. I don't care I just prefer for it to be consistent. If you prefer to omit them then feel free to change the rule in .sass-lint:
      ```yaml
        leading-zero:
          - 2
          -
            include: true # change to false to allow .5rem and disallow 0.5rem
      ```
   * Can we remove the commented out styles?

**Comment by lschatzkin** [comment]
Weird, I had added that line to assets.rb -- I'll put it back.
Good point about underscores.
Thanks for the note about leading zeros.

File: .sass-lint.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ rules:
     -
       include: true
 
-  nesting-depth: 2
+  nesting-depth: 3
```
Comment (wioux) [inline .sass-lint.yml]: It's weird but the 2 here isn't actually the maximum nesting depth, it specifies that a failed nesting-depth check is an error (they use 1 for warning and 2 for error).

For an example of changing the options for a rule look at the config for indentation above. The options for nesting-depth are documented here: https://github.com/sasstools/sass-lint/blob/develop/docs/rules/nesting-depth.md

Comment (lschatzkin) [reply .sass-lint.yml]: Thanks for the quick tutorial. That's what I get for guessing! A classic example RTFM.

**Pull request merged**